source/docs/articles/users/access-management.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Access Management for Site Owners and Administrators
+description: Learn how to remove a team member or outside resource from a site.
+keywords: remove team member, delete team member, delete user
+---
+
+When a person with access to your site(s) on the platform leaves the company or project, it is important to immediately remove them from the team so that they no longer have access to make changes to your site.
+
+After a user leaves, we recommend you:
+
+- Delete or block the user's account in [Drupal](https://www.drupal.org/node/627158) or [WordPress](https://codex.wordpress.org/Users_Users_SubPanel).
+- Remove the user from the team and/or organization in the Pantheon Dashboard.
+- Change any account passwords the user may have had access to.
```

Comment:
I would specifically say *shared* account passwords. Any individual ones for the user should be handled by blocking the access.